tests/unittest/_torch/thop/test_scaled_mm.py

```diff
@@ -60,8 +55,10 @@ def test_fp8_scaled_mm(output_dtype, m, k_n):
         out_dtype=output_dtype,
     )
     # set pytorch's cublas workspace size to 32MB to be aligned with trtllm
+    # if anywhere else calls torch's cublas op, the static workspace size will
```

Comment:
Can you explain why the test was failing before?

Reply:
In pipeline test, the test in test_fp8_linear will run first and the workspace size will be set to 1MB, which is a static value [here](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/cuda/CUDABlas.cpp#L220). For we don't launch new process, this value will also be applied to thie test.

Reply:
Can you add a comment on the splitk thing?

Reply:
Sure, done.